from_cpython/Lib/test/test_complex.py

```diff
@@ -440,7 +438,7 @@ def __rmod__(self,other):
                 for y in test_values:
                     a = 'x %s y' % op
                     b = 'y %s x' % op
-                    self.assertTrue(type(eval(a)) is type(eval(b)) is xcomplex)
+                    # self.assertTrue(type(eval(a)) is type(eval(b)) is xcomplex)
```

Comment:
This is a bug / missing feature in our binop handling; usually the rule is you try `lhs.__pow__(rhs)` and then try `rhs.__rpow__(lhs)`, *except* if rhs is a subclass of lhs, and then you reverse the order.  We don't support that reverse-order case, and I don't think it's specific to complex or pow.  It'd be great if you wanted to look into it (we would want to rewrite it, which could be a good intro to that stuff), but I think for this PR it's fine to comment out the check.